core/src/main/java/hudson/model/Run.java

```diff
@@ -2422,7 +2422,7 @@ public String getEntryID(Run e) {
 
     private static class DefaultFeedAdapter implements FeedAdapter<Run> {
         public String getEntryTitle(Run entry) {
-            return entry.getDisplayName()+" ("+entry.getBuildStatusSummary().message+")";
+            return entry.getFullDisplayName();
```

Comment:
Comparing with #2845 RSS feed will be with project name but not build status. This is not enough.
We need to see project name (for rss with multiple projects, what #2845 broke), build name and build status like it was before.